README.md

```diff
@@ -104,5 +98,12 @@ Since OpenShift V3 does not provide a git repository out of the box, you can con
 5. Paste your copied webhook url provided by OpenShift - Thats it!  
 6. After you save your webhook, if you refresh your settings page you can see the status of the ping that Github sent to OpenShift to verify it can reach the server.  
 
+#### Enabling the Blog example
+In order to access the example blog application, you have to remove the
+`public/index.html` which serves as the welcome page and rebuild the application.
+Another option is to make a request directly to `/articles` which will give you access to the blog.
+
+The username/pw used for authentication in this application are openshift/secret.
```

Comment:
"The username and password used for authentication in this application are openshift/secret."

(And does this actually work? Are these values hardcoded somewhere?)